docs/installation/local.md

```diff
@@ -63,6 +60,38 @@ brew install python@3
 brew install libmagic
 ```
 
+## Using pip and virtualenv
+
+Open a terminal and enter the following commands to setup a virtual environment
+
+```sh
+virtualenv -p python3 venv
+source venv/bin/activate
```

Comment:
Make it .venv

Reply:
@iamareebjamal done. please review again :+1: